README.md

```diff
@@ -54,10 +55,30 @@ sudo apt install python3-dev libpq-dev
 8. Now run `python manage.py runserver`. Go to http://127.0.0.1:8000/ in your browser. You should be able to access our project.
 
 ## Setup Synchronization (Optional)
+
+### SymmetricDS
 _Please make sure there is no whitespace in the path to the cloned repository. SymmetricDS doesn't work with a path with whitespaces._
 
 To setup SymmetricsDs on your machine, click [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/blob/master/symmetricds/README.md) In case you are on windows, please also refer to the wiki on the topic by going [here.](https://github.com/gcivil-nyu-org/spring2020-cs-gy-9223-class/wiki/Setting-up-SymmetricDS-on-Windows) Windows support is not streamlined and is very error prone but this should be enough to get you started. In case you want to help us make this better please refer below as to how you can contribute to the project.
 
+### Heroku Deployment
+1. Install Heroku CLI
+   1.  Instructions can be found here:https://devcenter.heroku.com/articles/heroku-cli
+       * For macOS: `brew tap heroku/brew && brew install heroku`
+       * For Ubuntu/Debian `curl https://cli-assets.heroku.com/install-ubuntu.sh | sh`
+       * For Arch Linux `yay -S heroku-cli`
+       * For Windows: Use this link address to install 64bit installer https://cli-assets.heroku.com/heroku-x64.exe OR Use this link address to install 32 bit installer https://cli-assets.heroku.com/heroku-x86.exe
+   2.  Verify your installation: Running `heroku --version`, output should be like `heroku/x.y.z`
+
+2. Heroku Deployment
+   1. Create an account in Heroku 
```

Comment:
this step could use a link to Heroku home page. or better: add a link to Heroku at the start of this section, in the explanation what this section is about.